weasyl/oauth2.py

```diff
@@ -92,3 +118,72 @@ def get_userid_from_authorization(scopes=['wholesite']):
 __all__ = [
     'get_consumers_for_user', 'revoke_consumers_for_user',
 ]
+
+
+def get_allowed_scopes(user_id):
```

Comment:
style nit: Although it's questionable as a decision, most of the rest of the codebase uses `userid`, and code should be as consistent as possible.